### ISSUE TITLE
`CategoricalDistribution` sampling returns incorrect type

### ISSUE THREAD

**Issue by gautams3** [pfnet/optuna#455]
If your issue is a **request for support in using Optuna**, please post it on [Stack Overflow].
Support is also available on `#general` channel of [chainer slack] and [chainer-jp slack].

[Stack Overflow]: https://stackoverflow.com/questions/tagged/optuna
[chainer slack]: https://bit.ly/join-chainer-slack
[chainer-jp slack]: https://bit.ly/join-chainer-jp-slack

On registering an issue, write precise explanations on how you want Optuna to be.

Bug reports must include necessary and sufficient conditions to reproduce the bugs.
More specifically, it is helpful for us if you include the following information:

**Conditions**
- Optuna version: 0.13.0
- Python version: 3.6.8
- OS: Ubuntu
- Machine Learning library to be optimized: [rl-baselines-zoo](https://github.com/araffin/rl-baselines-zoo)

**Code to reproduce**

```
# python code
import optuna
study = optuna.create_study(sampler=optuna.samplers.random)
possible_vals = optuna.distributions.CategoricalDistribution(choices=('auto', 1., 2., 3., 4.))
sampled_val = study.sampler.RandomSampler().sample_independent(study, 'my_trial', 'my_param', possible_vals)

print(sampled_val, type(sampled_val)) # returns numpy.str_ even when a float could be sampled
```

**Error messages, stack traces, or logs**
```
# stack traces
```

**Problem code**
optuna.samplers.random.py::sample_independent() has the offending line
```
        elif isinstance(param_distribution, distributions.CategoricalDistribution):
            choices = param_distribution.choices
            return self.rng.choice(choices) # should be choices[self.rng.choice(len(choices))] 
```
Thank you for your time!

**Comment by sile** [comment]
Thank you for reporting this problem.

I think this is the same problem with #448 and has been resolved by #449 .
The next release of Optuna will include the patch (the next version will be released within a few weeks).

**Comment by gautams3** [comment]
OK I see that. I'll close this issue.
